.ruby-version

```diff
@@ -1 +1 @@
-2.6.
+3.0.
```

Comment:
Hi @iamdeit!  For now, I just wanted to make tests run in the same version as SDS, for more of a guarantee that it will be compatible when SDS & DSS load it; since this is loaded as a gem, it doesn't actually get run in this specified version, this `.ruby-version` is only for development and testing.  So the short answer is: same reason we haven't gone straight to 3.2.0 everywhere else, which is that we haven't had time to migrate all services.  Though we'll probably go to 3.1.3 first, since I'm not usually inclined to be on the bleeding edge and use .0 patch-level releases before any fixes have gone out on a minor version bump. 😅

Reply:
Understood, thank you!